### PR TITLE
configurable permission in tx_wrapper

### PR DESCRIPTION
**Pull request by taokayan**
solve #458 

- EOS permission configurable
- expire sec configurable (default to 300s, reduced from 3000s)
- push_tx logging with transaction counter & latency, better for performance monitoring and trouble shooting
- gasPrice cache timeout increase from 0.5s to 1s
- simplified gasPrice result logging, only log the price instead of the json response.

### CODE REVIEW

File: peripherals/tx_wrapper/index.js [hunk 1]
```diff
@@ -29,6 +29,11 @@ if (!process.env.EOS_SENDER) {
   process.exit();
 }
 
+if (!process.env.EOS_PERMISSION) {
+  console.log("Missing EOS_PERMISSION in .env file!");
```
Comment (arhag) [inline peripherals/tx_wrapper/index.js:33]: I was okay with it defaulting to `active` if not specified, but I can actually like it being explicit more.

Please update the README to capture the new `EOS_PERMISSION` and `EXPIRE_SEC` variables.

Comment (taokayan) [reply peripherals/tx_wrapper/index.js:33]: updated [local_testnet_deployment_plan.md](https://github.com/eosnetworkfoundation/eos-evm/pull/503/commits/67d1ae8d8a424cb06a14864e27b9b56606f69ea3), please check again. thanks!

Comment (arhag) [reply peripherals/tx_wrapper/index.js:33]: Oh. I was referring to the README in peripherals/tx_wrapper.

Comment (taokayan) [reply peripherals/tx_wrapper/index.js:33]: Modified README. Please check it again. thanks!

**Review by arhag — APPROVED** [review]

**Pull request merged**